frontend/encore/simple-example.rst

```diff
@@ -82,7 +82,7 @@ your layout. In Symfony, use the ``asset()`` helper:
 
 .. code-block:: twig
 
-    {# templates/base.html.twig #}
+    {# app/Resources/views/base.html.twig #}
```

Comment:
both are valid for `3.4` though.

Reply:
`symfony new my_project 3.4` comes with `app/Resources/views/base.html.twig`, and the docs reference `app/Resources/views/` a lot.

It makes more sense to me. Don't you think?

Reply:
Sure! for consistency :+1: